spec/models/seed_uri_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'spec_helper'
+
+RSpec.describe SeedUri do
+  describe 'validation' do
+    let(:seed) { described_class.new(uri: uri) }
+    subject { seed.valid? }
+
+    context 'when it is valid' do
+      let(:uri) { 'http://www.slac.stanford.edu/' }
```

Comment:
Would suggest some additional tests of when valid, including:
* timemap URL (e.g., `https://swap.stanford.edu/timemap/link/http://blog.sina.com.cn/txxwjy/`)

Reply:
@justinlittman that is not something that would ever be in this table.  For that request the seed would be "http://blog.sina.com.cn/txxwjy/"